### PR TITLE
Lu saturday add .gitignore

### PR DESCRIPTION
**Pull request by kaixin2017**
Please see if added .git-ignore correctly, thank you!

### CODE REVIEW

File: Health-Application-Saturday/.gitignore [hunk 1]
```diff
@@ -35,3 +35,6 @@ out/
 
 ### VS Code ###
 .vscode/
```
Comment (aokuonghae) [inline Health-Application-Saturday/.gitignore:37]: Almost there. 
You want to add .idea the git ignore.

File: Health-Application-Saturday/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,244 @@
+root = true
+
+[*]
```
Comment (aokuonghae) [inline Health-Application-Saturday/.editorconfig:3]: Did you mean to add an editor config file? I want to make sure you understand what it is what it is doing before being merged in.

**Review by aokuonghae — CHANGES_REQUESTED** [review]
I left two comments. You will want to remove the  .idea folder from the remote branch. Refer to this stackoverflow:
https://stackoverflow.com/questions/11124053/accidentally-committed-idea-directory-files-into-git/11124105
Second comment.  
This is why you use branches :)

**Review by yildizbill — APPROVED** [review]

**Review by yildizbill — APPROVED** [review]

**Review by yildizbill — APPROVED** [review]

**Review by dsfackrell — APPROVED** [review]

File: Health-Application-Saturday/.editorconfig (hunk 2 above)
Comment (kaixin2017) [reply Health-Application-Saturday/.editorconfig:3]: I didn't mean to add an editor config file, this was done by accident while I was trying to add git ignore, I looked up it is for consistent code formatting, if not needed, we could set root=false right? Thank you for catching that.

**Review by KellyReedKC — APPROVED** [review]

File: Health-Application-Saturday/.editorconfig (hunk 2 above)
Comment (aokuonghae) [reply Health-Application-Saturday/.editorconfig:3]: I just want to make sure you were aware. If you want it in there, that is fine.

File: Health-Application-Saturday/.gitignore (hunk 1 above)
Comment (aokuonghae) [reply Health-Application-Saturday/.gitignore:37]: @kaixin2017 Add in the .idea to the ignore and remove the .idea from the remote and you should be good to go on merging

**Review by aokuonghae — APPROVED** [review]

**Pull request merged**